server/portal/apps/jupyter_mounts/api/views.py

```diff
@@ -5,7 +5,7 @@
 from django.utils.decorators import method_decorator
 from django.http import JsonResponse
 from portal.views.base import BaseApiView
-from portal.apps.projects.managers.base import ProjectsManager
```

Comment:
does this complete the migration from `ProjectsManager` to to `shared_workspace_operations`? Can ProjectsManager be dropped?

I just see change_project_role related stuff that i think might not be used, right?

if so, would that be in the PR or a follow-on Jira issue?

Reply:
@jarosenb ?

Reply:
Not yet, we won't be able to drop this code until all workspaces in legacy portals have been migrated.

Reply:
ah, yes, that makes sense! thanks.